tests/RobotFramework/tests/cumulocity/software.robot

```diff
@@ -26,7 +26,7 @@ tedge-agent should terminate on SIGINT while downloading file
     Execute Command                          chmod 777 /root
     Execute Command                          tedge config set tmp.path /root
     Restart Service                          tedge-agent
```

Comment:
Forgot about one thing: if we do throttling and don't have to download big files that don't fit in tmpfs, then we can remove this

Reply:
Yes I think it is worth simplifying the test to remove the custom tedge config (due to the previous constraints of the tmpfs).

Reply:
I've updated the test now to remove the custom tmp location, and I also updated some of the comments in the test which are no longer applicable.